src/DBConnection.hpp

```diff
@@ -1,14 +1,30 @@
 #pragma once
 
+#include <string>
+
+struct Info
+{
+	Info() : serverVersion(""), protocolVersion(0) {};
```

Comment:
If you wrote ctor please write and destructor too and vice versa.

Reply:
Missed destructor added